charts/example-configurations/minikube-values-gitlab.yaml

```diff
@@ -72,35 +72,18 @@ notebooks:
       type: gitlab
       gitlab:
         ## Put Application Id and Application Secret here!
-        clientId:
-        clientSecret:
-        callbackUrl: '""'
-
-gateway:
-  gatewayEnv: development
-  secretKey: 5495ef917488a19d1d4106452c899f5824972acd87edce11fd65e7956bea3bb0
-  gitlabUrl: https://gitlab.com
-  ## Put Application Id and Application Secret here!
-  gitlabClientId:
-  gitlabClientSecret:
-  jupyterhub:
-    clientId: *gatewayClient
-    clientSecret: *gatewaySecret
-
-runner:
-  enabled: true
-
-ui:
-  gitlabUrl: https://gitlab.com
+        clientId: "12345"
+        clientSecret: "12345"
+        callbackUrl:
 
 graph:
   jena:
     admin:
-      password: 94a635b40afc2cba
+      password: &sparql-password 94a635b40afc2cba
 
   ## if running gitlab on minikube, set on cmd line with --set graph.gitlab.url
-  gitlab:
-    url: https://gitlab.com
+  # gitlab:
```

Comment:
Why this change? I'd assume that `https://gitlab.com` is the correct value here.

Reply:
oops thanks for catching that. Probably a left-over from some experiment. I also apparently removed some other values which I've now put back.